cli/args/mod.rs

```diff
@@ -1171,7 +1181,11 @@ mod test {
     let config_specifier =
       ModuleSpecifier::parse("file:///deno/deno.jsonc").unwrap();
     let config_file = ConfigFile::new(config_text, &config_specifier).unwrap();
-    let actual = resolve_import_map_specifier(None, Some(&config_file));
+    let actual = resolve_import_map_specifier(
+      None,
+      Some(&config_file),
+      &std::env::current_dir().unwrap(),
```

Comment:
Maybe this should just provide something like `&PathBuf::from("/")` since it wont' ever be used?